i18n/nl/docusaurus-plugin-content-docs/current/staff/coaches/gamification.md

```diff
@@ -0,0 +1,84 @@
+---
+sidebar_position: 3
+---
+
+# Gamification
+
+|                       |                          |
+|-----------------------|--------------------------|
+| **Optioneel**         | Ja                       |
+| **Standaard**         | FULL                     |
+| **Feature Toggle**    | Gamification             |
+| **Ingeschakeld door** | Locatie Administrator    |
+| **Instellingen**      | Uit, FULL, LEVEL_XP_ONLY |
+|                       |                          |
+
+### Beschrijving
+
+We hebben een aantal gamification-functies om gebruikers te inspireren meer met de inhoud bezig te zijn.
+
+## Level & XP
+
+![Level- en XP-balk](/img/staff/coaches/gamification/xp-bar.png)
+
+Gebruikers kunnen experience points (XP) verzamelen van verschillende [acties](#welke-acties-xp-opleveren).
+Zodra de gebruiker genoeg XP heeft bereikt om een level omhoog te gaan, wordt er een level-up scherm getoond en heeft de
+gebruiker een extra level behaald.
+Hierna wordt de hoeveelheid XP die nodig is voor een level-up verhoogd.
+
+![Level-up scherm](/img/staff/coaches/gamification/level-up.png)
+
+### Welke acties XP opleveren
+
+We berekenen de hoeveelheid XP intern, we specificeren hier niet de exacte hoeveelheid XP, alleen een weergave.
+
+| Actie                      | XP-opbrengst |
+|----------------------------|--------------|
+| Reviews                    | Laag         |
+| Theorie-oefening afgerond  | Laag         |
+| Praktijk-oefening afgerond | Middel       |
+| Selfdiagnostic voltooid    | Middel       |
+| Project afgerond           | Hoog         |
+
+**Goed om te weten:**
+
+- De XP-opbrengst van theorie- en praktijkoefeningen verandert met hun moeilijkheidsgraad. Een hogere moeilijkheidsgraad
+  en tijdsindicatie leidt tot meer XP.
+- Als een oefening is heropend of herstart, wordt dezelfde hoeveelheid XP die de oefening opleverde, verwijderd uit het
+  studentenprofiel.
+
+## Scorebord
```

Comment:
Noemen we dit niet nog steeds leaderboard in het nederlands?

Reply:
Ja, ik twijfelde